src/App.tsx

```diff
@@ -18,23 +18,29 @@ const RootStack = createNativeStackNavigator({
         title: "Home",
       },
     },
+    RestaurantList: {
+      screen: RestaurantList,
+      options: {
+        title: "Find a restaurant",
+      },
+    },
     OrderCreate: {
       screen: OrderCreate,
       options: {
         title: "Order",
       },
-    },
-    RestaurantList: {
-      screen: RestaurantList,
-      options: {
-        title: "Find a restaurant",
+      initialParams: {
+        restaurantId: "placeholder Restaurant",
       },
     },
     RestaurantDetails: {
       screen: RestaurantDetails,
       options: {
         title: "Details",
       },
+      initialParams: {
+        restaurantId: "placeholder id",
+      },
```

Comment:
We should only have Home and Restaurants tabs in the navigation tab bar.

The Restaurants tab should be a stacked navigation like: States -> Cities -> Restaurant list/map -> Restaurant detail -> Order